build-configs/configs/BuildConfigType.js

```diff
@@ -49,7 +49,8 @@ export type BuildConfigType = {|
   lightTheme: ThemeType,
   darkTheme: ThemeType,
   iconSet: string,
-  development?: boolean,
+  development: boolean,
+  sentry: boolean,
```

Comment:
Shuldn't this be a part of featureFlags?

Reply:
Still relevant

Reply:
done